java/arcs/core/host/EntityHandleManager.kt

```diff
@@ -170,7 +170,7 @@ class EntityHandleManager(
                 type = CollectionType(EntityType(entitySpec.SCHEMA)),
                 mode = ReferenceMode
             )
-        ).activate()
+        ).activate(activationFactory)
```

Comment:
Isn't the defaultActivation factory just an embedded (in memory) one? So does this mean our E2E tests on Android recently weren't testing the StorageService?

Reply:
Yes that's correct :(

Reply:
BUT now they are, and they continue to pass :-)

Reply:
BTW, this issue was only introduced yesterday when I merged the Handle unification PR. So *before* that PR, tests were using Android ServiceStore. This has only been a brief issue.